### PR TITLE
deps: make update_deps.py the same across all repos

### PR DESCRIPTION
**Pull request by lunarpapillo**
Matching update_deps.py from Khronos/Vulkan-Tools, Khronos/Vulkan-Loader,
Khronos/Vulkan-ValidationLayers, LunarG/VulkanSamples, and
LunarG/VulanTools.

### CODE REVIEW

**Comment by mark-lunarg** [comment]
This is failing the commit message format checks -- see the results for Travis.

**Comment by lunarpapillo** [comment]
Weird that Travis shows passed, though the details show the failure... is this expected?

**Comment by lunarpapillo** [comment]
Also weird that Travis passed on all the other repos... do we do different checks for different repos?

**Comment by mark-lunarg** [comment]
No, it is expected -- it's currently an 'allowed' failure, but we do use it to communicate commit message standards when necessary.  And the checks do vary from repo to repo.  VVL benefits from substantially more community contribution and is more visible, hence the need for more policing to keep things matchy-matchy.

File: scripts/update_deps.py [hunk 1]
```diff
@@ -442,11 +442,13 @@ def CMakeBuild(self):
...
             num_make_jobs = multiprocessing.cpu_count()
-            if 'MAKE_JOBS' in os.environ:
+            env_make_jobs = os.environ.get('MAKE_JOBS', None)
+            if env_make_jobs is not None:
                 try:
-                    num_make_jobs = min(num_make_jobs, int(os.environ['MAKE_JOBS']))
+                    num_make_jobs = min(num_make_jobs, int(env_make_jobs))
                 except ValueError:
-                    pass
+                    print('warning: environment variable MAKE_JOBS has non-numeric value "{}".  '
```
Comment (mark-lunarg) [inline scripts/update_deps.py]: How about 'WARNING:  Env....'?  Remind me to buy you some capital letters! 

This all looks fine to me!

**Pull request merged**